### ISSUE TITLE
Assembly with name 'Microsoft.Windows.MixedReality.DotNetWinRT.Editor' already exists 

### ISSUE THREAD

**Issue by CDiaz-MS** [microsoft/MixedRealityToolkit-Unity#6737]
## Describe the bug

The following errors show up after the import of the latest test Foundation package
```
Assembly with name 'Microsoft.Windows.MixedReality.DotNetWinRT.Editor' already exists (Assets/MixedRealityToolkit.Providers/WindowsMixedReality/DotNetAdapter/Plugins/netstandard2.0/Microsoft.Windows.MixedReality.DotNetWinRT/Editor/Microsoft.Windows.MixedReality.DotNetWinRT.Editor.asmdef)
```

```
Assembly with name 'Microsoft.Windows.MixedReality.DotNetWinRT.Editor' already exists (Assets/MixedRealityToolkit.Providers/WindowsMixedReality/DotNetAdapter/Plugins/net46/Microsoft.Windows.MixedReality.DotNetWinRT/Editor/Microsoft.Windows.MixedReality.DotNetWinRT.Editor.asmdef)
```
## To reproduce
1. Import the latest Foundation test package

## Your Setup (please complete the following information)

- Unity Version: 2018.4.11f1
- MRTK Version:  v2.2

**Comment by davidkline-ms** [comment]
it appears that the dotnetadapter folder may gave gotten incorrectly checked into the repository

**Comment by davidkline-ms** [comment]
the dotnetadapter folder was not the cause. it appears that having the repository's manifest configured to auto-install msbuild for unity was causing the DotNetWinRT package to be bundled into the CI generated unitypackages

**Comment by davidkline-ms** [comment]
resolved with #6740 and #6754
